forest/bonsai/tests/test_simulate_gps_data.py

```diff
@@ -271,3 +277,169 @@ def test_get_path_distance(coords1, coords2, directions1, mocker):
 def test_get_path_close_locations(coords1, coords3):
     """Tests case distance of locations is less than 250 meters."""
     assert len(get_path(coords1, coords3, "foot", "mock_api_key")[0]) == 2
+
+
+@pytest.fixture
+def random_path(coords1, coords2, directions1, mocker):
+    # path returned from running get_path function
+    # starting from coords1 and ending on coords2
+    # with car as transport
+
+    return np.array(
```

Comment:
Thanks. Would it be possible to reuse the path from `directions1` fixture?

Reply:
Thanks! I am wondering if it would make sense to change `get_path()` to accept directions as an argument. This way no mocking would be required in tests and it could be reused more easily (including in this fixture). Thoughts?

Reply:
I don't know. Say that we have directions as an argument, then in the places where we call `get_path()` we would have to write the code preceding the api call and the api call every time we run `get_path()`. When testing that code wouldn't we have to use the mocking then, thus the mocking is gonna be used in the tests either way? Also the function is basically just a wrapper around the api call.

Reply:
Right, the openrouteservice API calls would be in higher level code which could be subject to integrated or functional tests (instead of unit tests) which can talk to outside services potentially. Anyway, I think this is good enough at the moment but may be we can revisit this later when we'll be merging the rest of the changes.

If you are curious, this is where this sort of architecture was discussed: https://www.youtube.com/watch?v=DJtef410XaM